sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/NetAppVolumeData.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.NetApp
+{
+    /// <summary>
+    /// A class representing the NetAppVolume data model.
+    /// Volume resource
+    /// </summary>
+    public partial class NetAppVolumeData : TrackedResourceData
+    {
+        /// <summary> Restoring. </summary>
+        public bool? IsRestoring { get; set; }
```

Comment:
I think we added it here because in the new api-version, we removed it from the schema?
If so, I think we should add the `[EditorBrowsable(EditorBrowsableState.Never)]` attribute on this.

Also since this property should still exist in previous api-version, we should add a
```
[CodeGenSerialization(nameof(IsRestoring), "<its wire name>"]
```
attribute to this class, and regenerate the library, to make sure the library still have the ability to include this property in the payload when sending request, and deserialize into this property when the response has this property.

Reply:
The change is that this is now a `readOnly` property,  incorrectly modeled in swagger and changed to match service behavior.

Reply:
then I think we should add the EditorBrowsable attribute only to the setter.
Could we add a comment (regular comment) around this property about the readonly stuff?

Reply:
Is it possible only on the setter ?  
I will add a comment